autopush-common/src/db/bigtable/mod.rs

```diff
@@ -115,12 +115,9 @@ impl BigTableDbSettings {
             .map_err(BigTableError::GRPC)
     }
 
+    // Health may require a different metadata declaration.
     pub fn health_metadata(&self) -> Result<Metadata, BigTableError> {
```

Comment:
Assuming this new health check works out I don't see the need to distinguish the exact same health vs regular metadata but I won't block on this.

Reply:
Honestly, I figured I'd leave it in case there's something else we may need to do that's specific to the health check.  (e.g. we need to specify some additional header or some alternate tag because we make the call so frequently or whatever.)

Reply:
YAGNI IMO